<ui version="4.0" >
 <class>StatusClass</class>
 <widget class="QWidget" name="StatusClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>492</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <family>Book Antiqua</family>
    <pointsize>18</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle" >
   <string>Status</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <widget class="QGroupBox" name="groupBox_satellite" >
     <property name="font" >
      <font>
       <family>Book Antiqua</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="title" >
      <string>Satellite</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" >
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4" >
          <item>
           <widget class="QLabel" name="agc" >
            <property name="minimumSize" >
             <size>
              <width>175</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>175</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text" >
             <string>AGC:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="minimumSize" >
             <size>
              <width>175</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>175</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text" >
             <string>SNR:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="minimumSize" >
             <size>
              <width>175</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>175</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text" >
             <string>Lock:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5" >
          <item>
           <widget class="QProgressBar" name="progressBar_siganl_Strength" >
            <property name="value" >
             <number>0</number>
            </property>
            <property name="textDirection" >
             <enum>QProgressBar::BottomToTop</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar_siganl_Quality" >
            <property name="value" >
             <number>0</number>
            </property>
            <property name="textDirection" >
             <enum>QProgressBar::BottomToTop</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4" >
            <property name="minimumSize" >
             <size>
              <width>175</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>175</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text" >
             <string>No</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Filme" >
     <property name="font" >
      <font>
       <family>Book Antiqua</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="title" >
      <string>Receive</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <item>
             <widget class="QLabel" name="label_5" >
              <property name="minimumSize" >
               <size>
                <width>175</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>175</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>Film Name:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" >
            <item>
             <widget class="QLabel" name="label_6" >
              <property name="minimumSize" >
               <size>
                <width>175</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>175</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>Received/Total:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7" >
            <item>
             <widget class="QLabel" name="label_8" >
              <property name="minimumSize" >
               <size>
                <width>175</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>175</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>Received Ratio:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8" >
            <item>
             <widget class="QLabel" name="label_filmName" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>N/A</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9" >
            <item>
             <widget class="QLabel" name="label_Receiver" >
              <property name="text" >
               <string>0/0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10" >
            <item>
             <widget class="QProgressBar" name="progressBar_Revceiver_length" >
              <property name="value" >
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_10" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" >
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9" >
          <item>
           <widget class="QLabel" name="label_9" >
            <property name="minimumSize" >
             <size>
              <width>175</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>175</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text" >
             <string>Status:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10" >
          <item>
           <widget class="QTextBrowser" name="textBrowser" />
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
